security-alert-notifier.rb

```diff
@@ -177,17 +181,33 @@ def fetch_repositories(http:, cursor: nil)
     github = GitHub.new
 
     if github.vulnerable_repos.any?
-      total_vulnerabilities = github.vulnerable_repos.sum { |repo| repo.alerts.length }
-      puts "WARNING: #{total_vulnerabilities} vulnerabilities in #{github.vulnerable_repos.length} repos"
+      if options[:repos].nil?
+        total_vulnerabilities = github.vulnerable_repos.sum { |repo| repo.alerts.length }
+        vulnerable_repos = github.vulnerable_repos.length
```

Comment:
Think we could rename this variable to be a bit clearer that it's a count of repos, not the repos themselves (especially given `github.vulnerable_repos` has the same name)?

Reply:
Ok. `repos` for this option is irritiating. Your first inspiration is good. Let's call it `filter`

You're right. `vulnerable_repo` isn't so good. `vulnerable_repo_count` puts it better.
```suggestion
        vulnerable_repo_count = github.vulnerable_repos.length
```